docs/feature_flag_source.md

```diff
@@ -145,16 +145,16 @@ spec:
         memory: 256Mi
 ```
 
-The relevant `FlagSourceConfigurations` are passed to the operator by setting the `openfeature.dev/flagsourceconfiguration` annotation, and is responsible for providing the full configuration of the injected sidecar.
+The relevant `FeatureFlagSources` are passed to the operator by setting the `openfeature.dev/featureflagsource` annotation, and is responsible for providing the full configuration of the injected sidecar.
```

Comment:
```suggestion
The relevant `FeatureFlagSources` are passed to the operator by setting the `openfeature.dev/featureflagsource` annotation, which provides the full configuration of the injected sidecar.
```
Did I interpret this correction?  Is it the annotation what provides the full configuration?  "is responsible for providing" is unnecessarily verbose -- cleaner to just say "it provides"